librz/util/str.c

```diff
@@ -3819,3 +3819,71 @@ RZ_API char *rz_str_version(const char *program) {
 }
 
 #undef RZ_STR_PKG_VERSION_STRING
+
+/**
+ * \brief Converts a raw buffer to a printable string based on the selected options
+ *
+ * \param  buf          The buffer to read
+ * \param  len          The length of the buffer
+ * \param  max_wrap_len The max line length when RZ_STR_STRINGIFY_WRAP is set
+ * \param  options      The options flags to follow to generate the output.
```

Comment:
Could you put `RZ_STR_STRINGIFY_WIDE16_LE` and the others in an enum? I know that the option will still be a int because you want to do `... | ... | ...` but at least you can specify what options is referring to.